README.md

```diff
@@ -9,39 +9,37 @@ Sailfish OS is a mobile operating system based on open source and developed by t
 ## Contents
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
-
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-* [Sailfish OS](#sailfish-os)
-* [Applications](#applications)
-* [Tips and Tricks](#tips-and-tricks)
-* [Instant Messaging](#instant-messaging)
-  * [In English](#in-english)
-  * [In German](#in-german)
-  * [In Finnish](#in-finnish)
-  * [In Spanish](#in-spanish)
-  * [In Italian](#in-italian)
-  * [In Turkish](#in-turkish)
-  * [In Chinese](#in-chinese)
-  * [In Russian](#in-russian)
-* [Academic](#academic)
-* [Related](#related)
-* [License](#license)
+- [Sailfish OS](#sailfish-os)
+- [Applications](#applications)
+- [Tips and Tricks](#tips-and-tricks)
+- [Instant Messaging](#instant-messaging)
+  - [In English](#in-english)
+  - [In German](#in-german)
+  - [In Finnish](#in-finnish)
+  - [In Spanish](#in-spanish)
+  - [In Italian](#in-italian)
+  - [In Turkish](#in-turkish)
+  - [In Chinese](#in-chinese)
+  - [In Russian](#in-russian)
+- [Academic](#academic)
+- [Related](#related)
+- [License](#license)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Sailfish OS
 
-* [Sailfish Wiki](https://sailfishos.org/about/) - Wiki about Sailfish OS
 * [Sailfish OS Community](https://sailfishos.org/community/) - About Sailfish OS
   community
-* [together.jolla.com](https://together.jolla.com/questions/) - The place to ask
+* [SailfishOS Forum](https://forum.sailfishos.org) - The place to ask
   questions
+  * [together.jolla.com](https://together.jolla.com/questions/) - Precedor of SailfishOS Forum, contains historical information
```

Comment:
My attempt to address https://github.com/sailfishos-community/awesome-sailfishos/pull/18#discussion_r719881080